specs/assets_overview/02_user_interface.rst

```diff
@@ -0,0 +1,59 @@
+=====================
+User interface
+=====================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`64`_  Assets overview  tschubotz  2019-01-22
+=====  ===============  =========  ==========
+
+.. _64: https://github.com/gnosis/safe/issues/64
+
+.. _`User interface`:
+
+
+#. `Main`_
+#. `User interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Assets overview screen
+-------------------------
+
+- Tapping the identicon or the address brings up the receive screen.
+- Tapping "Add token" brings up the manage tokens screen.
+- Amounts should be abbreviated / truncated as follows:
+    - Cut off after the 5th decimal, no matter how many decimals there
+      are: 0.12345ETH
```

Comment:
I'm against rounding, since rounding could imply that the user has (slightly) more funds than we display. Would like to prevent that. That's why I wrote "cut off". :)